.changeset/shy-cobras-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': major
+---
+
+The return type of the `.start(...)` method of actor behaviors now returns `void`.
```

Comment:
I reported the need for this changeset [here](https://github.com/statelyai/xstate/pull/3743#discussion_r1121893328) but I thought that I was looking at `Interpreter` while this was just in `StateMachine`. So I actually think that it's OK to skip it. It's also OK to keep it but we will have to remove/clean it up after releasing the new alpha release - since this one won't be relevant for the final v5 release.

```suggestion
---
---
```